### PR TITLE
Form updates + other components (radio-group, button)

### PR DESCRIPTION
**Pull request by bedakb**
Prettier applied some formatting, so that thing is in the diff as well.

### CODE REVIEW

File: src/radio-group/radio-group.tsx [hunk 1]
```diff
@@ -18,18 +18,22 @@ const RadioGroup: React.FunctionComponent<Props> = (props) => {
...
-                        value={option.value}
-                        aria-checked={option.value === props.value}
-                        checked={option.value === props.value}
-                        onChange={onChange}
-                        data-qa={`radio-${props.id}-${index}`}
-                    />
-                    <label htmlFor={props.id}>{option.label}</label>
+                <div key={index} className={`${props.className || ''}`}>
+                    <label className={`radio`}>
+                        {' '}
```
Comment (efekaptan) [inline src/radio-group/radio-group.tsx]: Better to give padding instead of this space?

File: src/form/form-group.tsx [hunk 2]
```diff
@@ -34,8 +33,14 @@ const FormGroup: React.FunctionComponent<Props> = (props) => {
                         data-qa={`form-group--toggle-${props.id}`}
                         className="form-group-toggle"
                     >
+                        {props.isExpanded ? (
+                            <span className="icon icon-expanded" />
+                        ) : (
+                            <span className="icon icon-collapsed" />
+                        )}
```
Comment (MostafaBalata) [inline src/form/form-group.tsx]: I would do it this way
```
<span className=`icon icon-${props.isExpanded?"expanded":"collapsed"}` />
```

File: src/radio-group/radio-group.tsx (hunk 1 above)
Comment (bedakb) [reply src/radio-group/radio-group.tsx]: Ups, that was not me, most likely prettier 😈

File: src/form/form.tsx [hunk 3]
```diff
@@ -117,7 +117,7 @@ class Form<MODEL extends object> extends React.Component<Props<MODEL>, State<MOD
             violations: this.props.violations || {},
             initialModel: this.props.model,
             isSubmitting: false,
-            expandedGroupId: this.props.children[0].id,
+            expandedGroupsIds: [this.props.children[0].id],
```
Comment (MostafaBalata) [inline src/form/form.tsx]: How can you be sure that children has always at least one element?

Comment (bedakb) [reply src/form/form.tsx]: Well If you don't provide one section at least, then you are using component incorrectly.

**Pull request merged**